src/plugins/discourse/mirror.js

```diff
@@ -473,6 +473,16 @@ export class Mirror implements DiscourseData {
         }
         offset += likeActions.length;
       }
+    };
+    for (const user of this.users()) {
+      try {
+        await addUserLikes(user);
+      } catch (e) {
+        console.warn(
```

Comment:
OK—in general I’d prefer to avoid logspam for cases that we know will be
hit, like this one, but I’m willing to wait and see how common this is.
Ideally, we could save “info-logging” statements like this to a separate
log that’s displayed only on request.